.github/workflows/ci.yml

```diff
@@ -5,13 +5,21 @@ on:
 jobs:
   validate:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
+    permissions:
+      id-token: write
+      contents: read
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3.7.0
         with:
-          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
       - run: ./script/ci
+      - uses: actions/upload-artifact@v3
+        with:
+          path: dist
+      - uses: guardian/actions-static-site@v2
+        with:
+          app: editions-card-builder
```

Comment:
It's non-obvious that this repository will be [deployed](https://github.com/guardian/riffraff-platform/pull/165) with the Riff-Raff project `deploy::editions-card-builder`. Worth adding something to the README?